PurchasesCoreSwift/Logging/Strings/Strings.swift

```diff
@@ -1,18 +1,27 @@
 //
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
 // Created by Andrés Boedo on 9/14/20.
-// Copyright (c) 2020 Purchases. All rights reserved.
 //
 
 import Foundation
 
-@objc(RCStrings) public class Strings: NSObject {
-    @objc public static let attribution = AttributionStrings()
-    @objc public static let configure = ConfigureStrings()
+class Strings {
```

Comment:
Based on this [conversation](https://github.com/RevenueCat/purchases-ios/pull/754#discussion_r692359866), `Strings` could be a `enum`, right?

Reply:
I just added a comment on another one of these classes. 
I think we should move them to `enum`s, but it's probably a good idea to do it in a different PR, so that the focus of this one can be public visibility + objc compatibility, and we can do all the strings changes in one pass

Reply:
💯

Reply:
#768